src/Widgets/EligibilityModal/components/Schedule/index.tsx

```diff
@@ -19,87 +19,91 @@ const Schedule: FC<{ currentPlan: EligibilityPlan }> = ({ currentPlan }) => {
   const intl = useIntl()
 
   return (
-    <div
-      className={cx(s.schedule, STATIC_CUSTOMISATION_CLASSES.scheduleDetails)}
-      data-testid="modal-installments-element"
-    >
-      <div className={cx(s.scheduleLine, s.total, STATIC_CUSTOMISATION_CLASSES.scheduleTotal)}>
-        <span>
-          <FormattedMessage id="eligibility-modal.total" defaultMessage="Total" />
-        </span>
-        <span>
-          <FormattedNumber value={total} style="currency" currency="EUR" />
-        </span>
-      </div>
+    <>
       <div
-        className={cx(s.scheduleLine, s.creditCost, STATIC_CUSTOMISATION_CLASSES.scheduleCredit)}
+        className={cx(s.schedule, STATIC_CUSTOMISATION_CLASSES.scheduleDetails)}
+        data-testid="modal-installments-element"
       >
-        {isCredit ? (
+        {(currentPlan?.payment_plan || []).map((installment, index) => (
+          <div className={s.scheduleLine} key={index}>
+            <span>
+              {isToday(installment.due_date * 1000) ? (
+                <FormattedMessage id="installments.today" defaultMessage="Aujourd'hui" />
+              ) : (
+                <FormattedDate
+                  value={installment.due_date * 1000}
+                  day="numeric"
+                  month="long"
+                  year="numeric"
+                />
+              )}
+            </span>
+            <span>
+              <FormattedNumber
+                value={priceFromCents(installment.total_amount)}
+                style="currency"
+                currency="EUR"
+              />
+            </span>
+          </div>
+        ))}
+      </div>
+      <div className={cx(s.summary)} data-testid="modal-summary">
```

Comment:
Can you add a new class in the className.const.ts ? It's for the merchant to customize this element if need be ?

Reply:
done

Reply:
You're rigth :) I reduced it